web-frontend/src/pages/Home/Home.tsx

```diff
@@ -31,11 +47,20 @@ export default function Home() {
     }
   }, [account, navigate]);
 
-  const unformattedBalance = account?.candidateBalance ?? '0';
-  const balance = parseInt(unformattedBalance);
-  const formattedBalance = formatStandard(balance, Unit.NanoMAS);
-  const address = account?.address ?? '';
-  const formattedAddress = maskAddress(address);
+  // Call handleResize once after component mounts to log formattedAddress on page load
+  useEffect(() => {
+    window.addEventListener('resize', handleResize);
```

Comment:
Please, don't!

Reply:
can you explain more why this is not good ?

Reply:
Let's sync in a call...

Reply:
And for those that were not in the call, what is the solution ? What is the bad practice ?

Reply:
it's not a scalable solution with a complexity of O(n) if we want to apply it elsewhere + useEffect hook is prone to introducing uncontrolled bugs that can be hard to identify. @SlnPons validated an alternative. All the information is in the pr description